src/ch04-00-understanding-ownership.md

```diff
@@ -1,7 +1,6 @@
-# Understanding Ownership
+# ทำความเข้าใจเกี่ยวกับ Ownership
 
-Ownership is Rust’s most unique feature, and it enables Rust to make memory
-safety guarantees without needing a garbage collector. Therefore, it’s
-important to understand how ownership works in Rust. In this chapter, we’ll
-talk about ownership as well as several related features: borrowing, slices,
-and how Rust lays data out in memory.
+Ownership เป็นลักษณะเฉพาะตัวที่สุดของ Rust ที่ทำให้ Rust สามาถรับประกัน memory safety ได้โดยที่ไม่ต้องมี
+ตัวจัดการขยะในหน่วยความจำ หรือ garbage collector ดังนั้นมันจึงสำคัญมากที่เราจะต้องเข้าใจว่า ownership ทำงานอย่างไร
```

Comment:
ใช้เป็น
`ตัวจัดการขยะในหน่วยความจำ(garbage collector - ผู้แปล)`

Reply:
I like the original version more. @kkrittipong what do you think?

Reply:
@dodadoa I like the word ฟีเจอร์ as well. Should I use the transliteration for this word?

Reply:
ผมคิดว่าใช้ทับศัพท์เป็นแค่ garbage collector ไปได้เลย ไม่ต้องแปลเป็นไทยว่า ตัวจัดการขยะในหน่วยความจำ ครับ

Reply:
I also think the same at first, I'm ok if we're gonna use transliteration for this as well. @dodadoa What do you think?

Reply:
agree with @kkrittipong also can be `การ์เบจคอลเลคเตอร์`.

Reply:
but garbage collector is best